Makefile

```diff
@@ -75,9 +76,18 @@ generate: controller-gen mockgen
 docker-build: test
 	docker build . -t ${IMG}
 
-# Push the docker image
-docker-push:
-	docker push ${IMG}
+# Build the image with podman
+podman-build: test
+	podman build . -t ${IMG}
+
+# Push the image with podman
+podman-push:
+	podman push ${IMG} --tls-verify=false
+
+test-integration: podman-build
```

Comment:
does that mean that test-integration is based on podman only?

Reply:
yes, this is because podman supports to push to insecure registries without much reconfiguration